linux_os/guide/system/software/integrity/software-integrity/aide/aide_scan_notification/rule.yml

```diff
@@ -54,9 +54,21 @@ references:
 ocil_clause: 'AIDE has not been configured or has not been configured to notify personnel of scan details'
 
 ocil: |-
+    Verify the operating system routinely checks the baseline configuration for unauthorized changes and notifies the system administrator when anomalies in the operation of any security functions are discovered.
+
     To determine that periodic AIDE execution has been scheduled, run the following command:
     <pre>$ grep aide /etc/crontab</pre>
     The output should return something similar to the following:
     <pre>05 4 * * * root /usr/sbin/aide --check | /bin/mail -s "$(hostname) - AIDE Integrity Check" root@localhost</pre>
     The email address that the notifications are sent to can be changed by overriding
     <pre><sub idref="var_aide_scan_notification_email" /></pre>.
+
+fix: |-
+    Configure the file integrity tool to run automatically on the system at least weekly and to notify designated personnel if baseline configurations are changed in an unauthorized manner.
+    The AIDE tool can be configured to email designated personnel with the use of the cron system.
+
+    Edit "/etc/crontab" and add or edit the following line:
+
+    <pre>05 4 * * * root /usr/sbin/aide --check | /bin/mail -s "$(hostname) - AIDE Integrity Check" root@localhost</pre>
+    The email address that the notifications are sent to can be changed by overriding
+    <pre><sub idref="var_aide_scan_notification_email" /></pre>.
```

Comment:
we can't use variables in fixtexts, the variables will be ignored by create_srg_exports.py

Reply:
The variables are handled. They just need to be explicitly set. I will set it.